.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: CI - Build
+
+on:
+  push:
+    branches:
+      #- master @TODO Enable master once testing is done!
+      - ci-deploy
+
+jobs:
+  build:
+    name: OTP ${{ matrix.otp }} | Elixir ${{ matrix.elixir }} | Node ${{ matrix.node }} | OS ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        otp: [22.0.7]
+        elixir: [1.9.1]
+        node: [12.8.0]
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-elixir@v1.1.0
+        with:
+          otp-version: ${{ matrix.otp }}
+          elixir-version: ${{ matrix.elixir }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install dependencies
+        run: |
```

Comment:
`env: prod` should be here?

Reply:
I mean:
```env:
          MIX_ENV: prod```

Reply:
Nopes, don't needed! It doesn't matter the `MIX_ENV` for this commands:
```
mix local.rebar --force
mix local.hex --force
mix archive.install --force .....phx_new.ez
mix deps.get
npm install
```

Reply:
The whole release build process is working fine!
New updates coming soon..  =]

Reply:
for the deps.get I mean, in the case you are using, in ex, `{:ex_doc, "~> 0.21", only: :dev, runtime: false},`

Reply:
Got it, you are right! It's a little improvement, will add it